.github/workflows/pull-request-checks.yaml

```diff
@@ -700,7 +700,9 @@ jobs:
       - name: Print ccache stats
         run: ccache -s
       - name: Run CTest and collect coverage statistics
-        run: cmake --build build --target coverage -- -j2
+        run: |
+          echo "lcov_excl_line = UNREACHABLE" > ~/.lcovrc
```

Comment:
Good catch! I'm not massively familiar with lcov (to my shame) so apologies if these are dumb questions... but:
* Does this exclude _only_ lines containing UNREACHABLE, or also any other lines following that in the same block/trace?
* Do we want/need to do something similar for branch coverage with `lcov_excl_br_line` ?

Reply:
Yes, it only affects exactly that one line that has "UNREACHABLE." There are ways to mark entire blocks (details in https://linux.die.net/man/1/geninfo), which we could consider if deemed useful.

I don't think we measure branch coverage at this time, but if needed we could certainly extend the configuration.